incubator/azuremonitor-containers/templates/omsagent-crd.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Is this CRD used by this chart? If not, where are the docs on how to use it?

Reply:
Its for internal use (health state). I will update readme to say so.

Reply:
added to readme.